### ISSUE TITLE
Not able to upload audio file on the app

### ISSUE THREAD

**Issue by purvi-ranawat** [OpenArchive/Save-app-android#139]
When I create the audio file or select the existing audio file and share the same on Save App , it is not getting uploaded. Instead only Save App opens and nothing happens after that

**Comment by foundscapes** [comment]
Phone: Pixel 3a
OS: Android 10
I was able to upload a short audio file from "Google Keep" by sharing directly from the Keep app.

Phone: Moto G
OS: Android 5.1
The same thing happened to me as Purvi when trying to share from "my sounds" app - clicked share to Save from within the app and Save opened but there was no file there.

**Comment by roseregina** [comment]
0.2.3-Beta-3
Galaxys7
Good Cloud (oademo) 

I was able to share from memo and upload successfully. I shared from files and uploaded an mp3 successfully as well.

0.2.3-Beta-3
MotoG
Good Cloud (oademo)

I was able to upload mp3s that I had downloaded three times, using the share approach. I was not able to upload via _My Sounds_ app because it only allows the option of sending a link, not sharing the file. _**Save**_ crashed multiple times while I was testing this and crash reports have been sent. 

I believe this issue is resolved, assuming that the crashes were unrelated.
